cpp/include/cudf/strings/strings_column_view.hpp

```diff
@@ -128,21 +131,22 @@ class strings_column_view : private column_view {
    * The offsets child must be used to properly address the char bytes.
    *
    * For example, to access the first character of string `i` (accounting for
-   * a sliced column offset) use: `chars_begin()[offsets_begin()[i]]`.
+   * a sliced column offset) use: `chars_begin(stream)[offsets_begin()[i]]`.
    *
    * @return Iterator pointing to the first char byte.
    */
-  [[nodiscard]] chars_iterator chars_begin() const;
+  [[nodiscard]] chars_iterator chars_begin(rmm::cuda_stream_view) const;
```

Comment:
the param is not used, so no name for it. since there is no name, doxygen will throw a warning for param without a name.
`warning: unexpected token in comment block while parsing the argument of command param`
When we skip this, doxygen doesn't mind.